gradle-plugin/CHANGELOG.md

```diff
@@ -6,7 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
-- Upgrade `com.android.tools.build:gradle` plugin to v3.3.2 (#41)
+
+## [0.2] - 2019-03-18
+* Upgrade `com.android.tools.build:gradle` plugin to v3.3.2 (#41)
```

Comment:
Thanks for updating the changelog! This should be under a `### Changed` header and the item below should be under an `### Added` header. i.e.:
```
## [0.2] - 2019-03-18
### Added
* Injection of custom maven repository for `application-services`.

### Changed
* Upgrade `com.android.tools.build:gradle` plugin to v3.3.2 (#41)
```
 Those are the two most common categories but if you wanted to find out more, I'm following the format of https://keepachangelog.com/en/1.0.0/